### PR TITLE
Adding small updates to reporting section

### PR DESCRIPTION
**Pull request by emmairwin**

### CODE REVIEW

**Review by Aaron-Junker — APPROVED** [review]

File: code-of-conduct.html [hunk 1]
```diff
@@ -72,7 +72,8 @@ <h3 class="h3">Scope</h3>
             </div>
             <div class="pb-6">
                 <h3 class="h3">Enforcement and Reporting</h3>
-                <p> We encourage all communities to resolve issues on their own whenever possible. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at opencode@microsoft.com.</p>
+                <p> We encourage all communities to resolve issues on their own whenever possible. Instances of abusive, harassing, or otherwise unacceptable behavior should be reported to the community leaders responsible for enforcement in a given project or to opencode@microsoft.com. If you are a Microsoft employee looking for support, please use the <a href="https://aka.ms/community-911-landingpage">Community 911 reporting process</a>.
```
Comment (jeffwilcox) [inline code-of-conduct.html:75]: should this link open in a new window or tab, with `target="_new"`, or self by default?

Comment (emmairwin) [reply code-of-conduct.html]: updated

**Review by jeffwilcox — APPROVED** [review]

**Comment by jeffwilcox** [comment]
Thank you!!

**Review by yehiamohamedyehia281 — COMMENTED** [review]
Thankyou

**Pull request merged**